### PR TITLE
fix: RMA on items added to order through swaps and claims

### PR DESCRIPTION
**Pull request by kasperkristensen**
**What**
- It was not possible to claim items that were added to an order through `swap.additional_items` and `claim.additional_items`.
- It was not possible to receive a return on items added to the order through `claim.additional_items`.

**How**

- Added relations `["swaps", "swaps.additional_items", "claims", "claims.additional_items", "swaps.additional_items.tax_lines", "claims.additional_items.tax_lines"]` to order retrieval in `/create-claim`
- Added relations `["claims", "claims.additional_items"]` to `ReturnService.receive`

**Testing**
- Integrations test for creating claims on items added to order as additional items on both swap and claim.
- Integrations tests for receiving a return on items added to order as additional items on both swap and claim.

### CODE REVIEW

**Comment by olivermrbl** [comment]
Can I get you to add a couple of integration tests for the fix? @kasperkristensen

**Comment by kasperkristensen** [comment]
@olivermrbl just bumping this one as it would be nice to get in 😄

File: packages/medusa/src/services/claim.js [hunk 1]
```diff
@@ -230,7 +230,35 @@ class ClaimService extends BaseService {
       let toRefund = refund_amount
       if (type === "refund" && typeof refund_amount === "undefined") {
         const lines = claim_items.map((ci) => {
-          const orderItem = order.items.find((oi) => oi.id === ci.item_id)
+          const allOrderItems = order.items
+
+          if (order.swaps && order.swaps.length) {
```
Comment (olivermrbl) [inline packages/medusa/src/services/claim.js:235]: very nit(non-blocking):
```suggestion
          if (order?.swaps?.length) {
```

File: packages/medusa/src/services/claim.js [hunk 2]
```diff
@@ -230,7 +230,35 @@ class ClaimService extends BaseService {
...
+                  it.shipped_quantity ||
+                  it.shipped_quantity === it.fulfilled_quantity
+                ) {
+                  allOrderItems.push(it)
+                }
+              })
+            }
+          }
+
+          if (order.claims && order.claims.length) {
```
Comment (olivermrbl) [inline packages/medusa/src/services/claim.js:248]: very nit(non-blocking):
```suggestion
          if (order?.claims?.length) {
```

**Review by olivermrbl — APPROVED** [review]
LGTM!

**Review by srindom — APPROVED** [review]
LGTM!

**Pull request merged**